### PR TITLE
added head function

### PR DESCRIPTION
**Pull request by rudk0**

### CODE REVIEW

File: src/head.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function head(arr) {
+    return arr[0];
```
Comment (AllDayAlone) [inline src/head.js:2]: Please handle the situation when somebody provides invalid argument (like null)

And write some test in `test/head.spec.js`

Comment (rudk0) [reply src/head.js]: Added tests and handling of invalid arguments

File: test/head.spec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const assert = require('assert');
+const head = require('../src/head');
+
+function test() {
+  
+  // success
+  assert.equal(head([1, 2, 3, 4]), 1);
+  assert.equal(head(['lol', 'kek']), 'lol');
```
Comment (AllDayAlone) [inline test/head.spec.js:8]: 🤔

File: test/head.spec.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  // bad arguments
+  assert.equal(head([]), 'Value is not valid');
```
Comment (AllDayAlone) [inline test/head.spec.js:11]: let's return undefined

**Pull request merged**